dockerfiles/docker-compose.yml

```diff
@@ -65,6 +65,8 @@ services:
       - cache
     environment:
       - DOCKER_NO_RELOAD
+      - RTD_EXT_THEME_ENABLED
```

Comment:
This one was already added :)

https://github.com/readthedocs/common/pull/217

Reply:
I'll rebase this before merging, I just wanted to make sure I didn't lose this change